### PR TITLE
deleted AddMealModal and renamed Schedule file

### PR DESCRIPTION
**Pull request by GuiFSs**
#71

### CODE REVIEW

File: src/components/Modals/AddMealModal.js [hunk 1]
```diff
@@ -1,73 +0,0 @@
-import React, { Component } from "react";
-import { Button, Modal, ModalHeader, ModalBody, ModalFooter } from "reactstrap";
-import { AvForm } from "availity-reactstrap-validation";
-import { MealForm } from "../MealPlanner/Form";
```
Comment (atherdon) [inline src/components/Modals/AddMealModal.js]: ok, what about this component?

File: src/components/Modals/AddMealModal.js [hunk 2]
```diff
@@ -1,73 +0,0 @@
-import React, { Component } from "react";
-import { Button, Modal, ModalHeader, ModalBody, ModalFooter } from "reactstrap";
-import { AvForm } from "availity-reactstrap-validation";
```
Comment (atherdon) [inline src/components/Modals/AddMealModal.js]: i think we should remove this package from our project for now too....

File: src/components/Modals/AddMealModal.js (hunk 1 above)
Comment (GuiFSs) [reply src/components/Modals/AddMealModal.js]: this is the file i just deleted

Comment (atherdon) [reply src/components/Modals/AddMealModal.js]: check the latest line here. my comment related to it

File: src/components/Modals/AddMealModal.js (hunk 2 above)
Comment (atherdon) [reply src/components/Modals/AddMealModal.js]: same here - check the last line

File: src/components/Modals/AddMealModal.js (hunk 1 above)
Comment (GuiFSs) [reply src/components/Modals/AddMealModal.js]: i'm sorry but, did you mean the others components that i updated?

Comment (atherdon) [reply src/components/Modals/AddMealModal.js]: MealPlanner/Form component should be removed too.

File: src/components/Modals/AddMealModal.js (hunk 2 above)
Comment (atherdon) [reply src/components/Modals/AddMealModal.js]: **Availity reactstrap validation** remove from project please

**Comment by atherdon** [comment]
please note that i updated some of our packages at package.json - so you'll need to pull that changes, before making a new commits to this PR

**Comment by GuiFSs** [comment]
i think it's ok now

**Comment by atherdon** [comment]
Ok, looks fine - did you also make fix for my 2 comments?

**Comment by atherdon** [comment]
Ok, got it - then i'm merging your changes

**Pull request merged**